### ISSUE TITLE
Does not write cfg (5808) bits

### ISSUE THREAD

**Issue by jkenneke** [klammerj/atfprog#2]
Thank you for creating this project! 

I just built the circuit, and everything works fine except the CFG bits are not being set properly (5805...).

JEDCMP reports an error, and the part does not work in the circuit. I don't have another reader around to double check what GALP is reading. But, since all the fuse bits are correct, this is a bit confusing.

Jon

**Comment by klammerj** [comment]
Yes. It's buggy. Sometimes it needed multiple writes to get them set.. sometimes a bit in the reset term would be stuck. I believe someone at the yahoo cosmac elf forum used a programming  voltage of 13V. maybe that's it.. or an actual bug in the code(It's all guesswork). I'd fix it if I knew what's wrong, but have used up my parts...
